Symfony/CS/Tokenizer/Token.php

```diff
@@ -367,6 +395,13 @@ public function isNativeConstant()
         return $this->isArray && in_array(strtolower($this->content), $nativeConstantStrings, true);
     }
 
+    public function isMagicConstant()
```

Comment:
sure thing :)
https://github.com/SpacePossum/PHP-CS-Fixer/commit/85732838eb4aa261387316931ea487345f7d0cb5
this way both lower and upper case for all magic constants are tested.. lets see what travis thinks..